lib/nifty_services/base_action_service.rb

```diff
@@ -17,23 +16,28 @@ def execute
             success_response
           else
             errors = action_errors
-            bad_request_error(errors) if errors.present?
+            bad_request_error(errors) unless errors.empty?
           end
         end
       end
     end
 
     private
+
     def action_errors
       []
     end
 
     def can_execute?
       unless user_can_execute_action?
-        return (valid? ? unprocessable_entity_error!(invalid_action_error_key) : false)
+        if valid?
```

Comment:
@viniciuszago I don't see why changing this is good too, one line expressions are made for this! This is a 80 line width expression..